README.md

```diff
@@ -1,8 +1,16 @@
 # Synergistic Software Platform for AI, Physics Simulations, and Experiments (Synapse)
 
+Synapse is a modular framework to build components of digital twins.
+It supports operators of machines and experiments with ML-assisted predictions, trained on a combination of continously measured and simulated data.
+Synapse embraces emerging [integrated research infrastructures](https://www.nersc.gov/what-we-do/computing-for-science/integrated-research-infrastructure), deploying a user-facing cloud service, using HPC/cloud compute, and exchanging modular components via container registries.
+
+At the moment, Synapse uses the NERSC Spin (control, dashboard), NERSC SuperFacilityAPI (simulation submission & ML training on PM), and NERSC container registry.
```

Comment:
I think NERSC writes "Superfacility API" usually (lower-case "f" and a white space between the two words): https://docs.nersc.gov/services/sfapi/.

Also, I would suggest that we replace "PM" with "Perlmutter", since maybe the "PM" abbreviation is not obvious to everyone.


```suggestion
At the moment, Synapse uses the NERSC Spin (control, dashboard), NERSC Superfacility API (simulation submission & ML training on Perlmutter), and NERSC container registry.
```